### PR TITLE
Video processing

### PR DESCRIPTION
**Pull request by visionjo**
Work in progress

### CODE REVIEW

**Comment by codezakh** [comment]
General comment:
pickle files might not be a good way of storing data.
1) they can run arbitrary python code on your system
2) they might not work across different versions of python/library

Perhaps we can switch to JSON or .npy files instead?

File: scripts/audio-processing/extract_wav_from_mp4_mp.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+def save_audio(f):
+    try:
+        clip = me.VideoFileClip(f[0])
+        clip.audio.write_audiofile(f[1])
+
+    except OSError:
+        print(f)
+
+
+# Step 1: Init multiprocessing.Pool()
```
Comment (codezakh) [inline scripts/audio-processing/extract_wav_from_mp4_mp.py:21]: Check out https://joblib.readthedocs.io/en/latest/, has nice convenicne funcs for doing this kinda stuff, sklearn uses it

**Comment by visionjo** [comment]
Okay. I like pickles to dump python objects. But I will do that for myself and consider other means of storage for you and public

File: scripts/audio-processing/extract_wav_from_mp4_mp.py (hunk 1 above)
Comment (visionjo) [reply scripts/audio-processing/extract_wav_from_mp4_mp.py:21]: oh i heard of this. I will check out and add a part of save actions

Comment (visionjo) [reply scripts/audio-processing/extract_wav_from_mp4_mp.py:21]: also, don't mind all the commented code. Next commits of these files that will be gone

File: scripts/feature-processing/extract_feature_arcface.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
+# Helper function for extracting features from pre-trained models
+## Download model weights on google drive:
+# https://drive.google.com/file/d/1pA6iZYQ2i8BaVNn4ngXaKeIf6v50tk1m/view?usp=sharing
```
Comment (codezakh) [inline scripts/feature-processing/extract_feature_arcface.py:3]: Nice!

**Pull request merged**